types/src/front/assistant/actions/process.ts

```diff
@@ -4,6 +4,7 @@ import {
   TimeFrame,
 } from "../../../front/assistant/actions/retrieval";
 import { ModelId } from "../../../shared/model_id";
+import { BaseAction } from "../../lib/api/assistant/actions";
```

Comment:
Must come from the root I beleive?

Reply:
https://github.com/dust-tt/dust/pull/5110 🙏🏻